bootstrap

```diff
@@ -121,10 +121,21 @@ rm(Path) ->
     ok.
 
 build_time() ->
-    {{Y, M, D}, {H, Min, S}} = calendar:now_to_universal_time(now()),
+    {{Y, M, D}, {H, Min, S}} = calendar:now_to_universal_time(rebar_now()),
     lists:flatten(io_lib:format("~4..0w~2..0w~2..0w_~2..0w~2..0w~2..0w",
                                 [Y, M, D, H, Min, S])).
 
+%% Even though we don't have -mode(compile), erl_lint:module/1 is run in
+%% escript, so we still have to silence the deprecation warning.
+-compile({nowarn_deprecated_function, [{erlang, now, 0}]}).
```

Comment:
I didn't use that because it's too broad and can hide warnings we want to see, but if the pre-18.x warning is a real concern, perhaps we should.

@ferd?